framework/src/setroubleshoot/server.py

```diff
@@ -116,6 +116,13 @@ def sighandler(signum, frame):
     import sys    
     sys.exit()
 
+def polling_failled_handler(signum, frame):
+    log_debug("received signal=%s" % signum)
+    if signum == signal.SIGALRM:
+        syslog.syslog(syslog.LOG_ERR, "/sys/fs/selinux/policy is in use by another process. Exiting!")
```

Comment:
It looks like this call of syslog.syslog() would happen before syslog.openlog(). It means that there would be log entry with python id instead of setroubleshoot id, e.g:

```
Mar 12 09:25:41 workstation python[26037]: ERROR
```
vs
```
Mar 12 09:26:04 workstation setroubleshoot[26037]: ERROR
```

Reply:
setroubleshootd calls syslog.openlog() immediately after import section (therefore your first case can only happen if RunFaultServer is used elsewhere).
But I can move the call in server.py as well if you want.